crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -147,6 +147,37 @@ impl ZombienetCommand {
 				}
 
 				spinner.stop(result);
+
+				// Check for any specified channels
+				if zombienet.hrmp_channels() {
+					let relay_chain = zombienet.relay_chain();
+					match RelayChain::from(relay_chain) {
+						None => {
+							log::error(format!("🚫 Using `{relay_chain}` with HRMP channels is currently unsupported. Please use `paseo-local` or `rococo-local`."))?;
+						},
+						Some(_) => {
+							use tokio::time::sleep;
+							let spinner = cliclack::spinner();
+							spinner.start("Connecting to relay chain to prepare channels...");
+							// Allow relay node time to start
+							sleep(Duration::from_secs(10)).await;
+							spinner.start("Preparing channels...");
```

Comment:
I haven't ran this code, but I assume that this second `spinner.start` simply replaces the original spinner message (3 lines above).

In this case, won't `spinner.set_message("...")` be more clear?

Reply:
Improved in 2e71e04b